blag/blag.py

```diff
@@ -396,7 +396,7 @@ def generate_feed(
         )
 
     with open(f"{output_dir}/atom.xml", "w") as fh:
-        feed.write(fh, encoding="utf8")
```

Comment:
Are you sure this actually changes something? I think `utf8` and `utf-8` are synonymous.

Reply:
Very good catch, thank you! I've fixed this and added a test for correct encoding in a separate PR.